staging/src/k8s.io/client-go/openapi/cached/groupversion.go

```diff
@@ -36,10 +52,56 @@ func newGroupVersion(delegate openapi.GroupVersion) *groupversion {
 	}
 }
 
+func (g *groupversion) SchemaPB() ([]byte, error) {
+	g.pbOnce.Do(func() {
+		g.pbBytes, g.pbErr = g.delegate.SchemaPB()
+	})
+
+	return g.pbBytes, g.pbErr
+}
+
+func (g *groupversion) SchemaJSON() ([]byte, error) {
+	g.jsonOnce.Do(func() {
+		g.jsonBytes, g.jsonErr = g.delegate.SchemaJSON()
+	})
+
+	return g.jsonBytes, g.jsonErr
+}
+
+func (g *groupversion) SchemaOpenAPI() (*spec3.OpenAPI, error) {
+	g.kubeOnce.Do(func() {
+		var data []byte
+		data, g.kubeErr = g.SchemaJSON()
+		if g.kubeErr != nil {
+			return
+		}
+
+		document := &spec3.OpenAPI{}
+		if g.kubeErr = json.Unmarshal(data, document); g.kubeErr != nil {
+			return
+		}
+
+		g.kubeDoc = document
```

Comment:
Why don't you use the delegate here (and below)?

Reply:
so that the raw bytes arent possibly fetched multiple times